app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaAuthorizedViewModel.kt

```diff
@@ -65,7 +65,7 @@ import javax.inject.Inject
 
 @HiltViewModel
 @Suppress("LongParameterList", "TooManyFunctions")
-class ImportMediaViewModel @Inject constructor(
+class ImportMediaAuthorizedViewModel @Inject constructor(
```

Comment:
Tiny code renaming for consistency with other classes.
```suggestion
class ImportMediaAuthenticatedViewModel @Inject constructor(
```